pkg/network/infraconfigurators/masquerade_test.go

```diff
@@ -27,6 +27,281 @@ const (
 	ipTables
```

Comment:
The commit message is cut

Reply:
The real issue was that the commit msg was too long; github showed it wrong.